CHANGES_NEXT_RELEASE

```diff
@@ -1 +1 @@
-
+- [cygnus-ngsi] Upgrade Dockerfile base image from centos8.4.2105 to Debian 11.2-slim
```

Comment:
What about other Dockefiles in this repo (cygnus-commons, cygnus-twitter, cygnus-ngsi-ld)?

If we don't want to invest effort in non-used docker images, then maybe a FIXME in the respective Dockerfiles is enough.

Reply:
Fixed in [12bc6ea](https://github.com/telefonicaid/fiware-cygnus/pull/2152/commits/12bc6eae106c6bbb38ecee076ea64b3a75c91a19)